env0/resource_environment.go

```diff
@@ -217,6 +217,11 @@ func resourceEnvironment() *schema.Resource {
 					Schema: getTemplateSchema("without_template_settings.0."),
 				},
 			},
+			"is_remote_backend": {
+				Type:        schema.TypeBool,
+				Description: "should use remote backend",
+				Optional:    true,
```

Comment:
for type boolean it's always recommended to set a default value...

Reply:
you then don't need to set IsRemoteBackend as a pointer.
In addition, you won't need the "tfschema..."